packages/reports/tests/acceptance/date-filter-test.js

```diff
@@ -43,29 +43,27 @@ module('Acceptance | date filter', function(hooks) {
 
     //Set low value
     await clickTrigger('.filter-values--dimension-date-range-input__low-value .ember-basic-dropdown-trigger');
-    await click($('.dropdown-date-picker__dropdown td.day:not(.old):not(.new):contains(17)')[0]);
-    await click('.dropdown-date-picker__apply');
+    await click($('button.ember-power-calendar-day--current-month:contains(4)')[0]);
 
     //Set high value
     await clickTrigger('.filter-values--dimension-date-range-input__high-value .ember-basic-dropdown-trigger');
-    await click($('.dropdown-date-picker__dropdown td.day:not(.old):not(.new):contains(19)')[0]);
-    await click('.dropdown-date-picker__apply');
+    await click($('button.ember-power-calendar-day--current-month:contains(5)')[0]);
 
-    assert.ok(!!$('.filter-values--dimension-date-range-input__low-value:contains(17)').length, 'The low value is set');
-    assert.ok(
-      !!$('.filter-values--dimension-date-range-input__high-value:contains(19)').length,
-      'The high value is set'
-    );
+    assert
+      .dom('.filter-values--dimension-date-range-input__low-value .dropdown-date-picker__trigger')
+      .includesText('04,', 'The low value is set');
+    assert
+      .dom('.filter-values--dimension-date-range-input__high-value .dropdown-date-picker__trigger')
+      .includesText('05,', 'The high value is set');
 
     await click('.navi-report__save-btn');
 
-    assert.ok(
-      !!$('.filter-values--dimension-date-range-input__low-value:contains(17)').length,
-      'The low value is still set after the report is saved'
-    );
-    assert.ok(
-      !!$('.filter-values--dimension-date-range-input__high-value:contains(19)').length,
-      'The high value is still set after the report is saved'
-    );
+    assert
+      .dom('.filter-values--dimension-date-range-input__low-value .dropdown-date-picker__trigger')
+      .includesText('04,', 'The low value is still set after the report is saved');
+
+    assert
+      .dom('.filter-values--dimension-date-range-input__high-value .dropdown-date-picker__trigger')
+      .includesText('05,', 'The high value is still set after the report is saved');
```

Comment:
Do we have any tests that moves one date operator to another?

Reply:
These?
https://github.com/yahoo/navi/blob/2bf75592f1015ed115e2a80247ebfec203d45e7b/packages/reports/tests/unit/components/filter-builders/date-time-test.js#L109-L365

Reply:
I was thinking from a UI perspective

Reply:
Nope I can add some though